isolator_injection_init.py

```diff
@@ -1100,6 +1100,7 @@ def main(ctx_factory=cl.create_some_context, user_input_file=None,
     current_cv = bulk_init(discr=discr, x_vec=thaw(discr.nodes(), actx),
                            eos=eos, time=0)
     current_state = make_fluid_state(current_cv, gas_model, init_temperature)
+    current_state = thaw(freeze(current_state, actx), actx)
```

Comment:
Does this approach have an advantage over the one I took here (and elsewhere)?

https://github.com/illinois-ceesd/drivers_y2-isolator/blob/e25a7bbd7a27dd6c803d7871c64afece30aa9bd5/isolator_injection_run.py#L670

Reply:
Not quite. Probably just an easier way to do the same thing?